added documentation for the rabbitmq message publisher (#470)
valtimo-platform/valtimo-documentation: getting-started/modules/core/outbox.md

#### Gradle dependency:

```groovy
dependencies {
    implementation "com.ritense.valtimo:outbox:${valtimo_version}"
}
```

## Configuration

> In order to use this module, the following configuration properties need to be added to the application properties. Please adjust values to match the environment:

Reviewer: ```suggestion
The outbox can be configured to better match the environment:
```
Because the `polling.rate` property is optional
